### ISSUE TITLE
Supercharger place is not recognized

### ISSUE THREAD

**Issue by 3isenHeiM** [adriankumpf/teslamate#3217]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I've recently charged at a Supercharger, but the place is not recognized by TM while being present in OSM.
Here is a copy of the map:
![afbeelding](https://github.com/adriankumpf/teslamate/assets/26417172/557d7354-dde8-4da6-a3aa-1143e83d4843)

### Expected Behavior

The place should be detected as "Zaventem Supercharger, Zaventem", but it's detected as "Weiveldlaan 29, Zaventem".

### Relevant log output

```shell
None
```

### Type of installation

Docker

### Version

v1.27.2

**Comment by SiNaPsEr0x** [comment]
some problem

**Comment by jheredianet** [comment]
> some problem, do you have some problem @jheredianet ?

No, I cannot reproduce that issue. The name of the detected place depends on the resolution of the gps point given by [nominatim.openstreetmap.org ](https://nominatim.openstreetmap.org/ui/search.html) service.

**Comment by JakobLichterfeld** [comment]
No update from OP since 6 months. Issue seems to be GPS related. You can even use as custom Geofence with wider radius there. --< close for the moment

No update from OP for 6 months. Problem seems to be GPS related. You can even use  a custom geofence with a larger radius. --> closed for now
